games-arcade/rocksndiamonds/patches/rocksndiamonds-3.3.1.2.patchset

```diff
@@ -1,4 +1,4 @@
-From ba88fe1ad26fc936fcfece46929b4b1eebb091c1 Mon Sep 17 00:00:00 2001
+From ba88fe1ad26fc936fcfece46929b2b1eebb091c1 Mon Sep 17 00:00:00 2001
```

Comment:
Sorry @dash102, I hope you won't stop claiming tasks because of my pickyness.
You are welcome to ignore those of my reviews which I tag as "not required but appreciated" ;-)